pennylane/qchem/observable_hf.py

```diff
@@ -117,9 +118,16 @@ def qubit_observable(o_ferm, cutoff=1.0e-12):
                 ops = ops + op[1]
                 coeffs = qml.math.concatenate([coeffs, qml.math.array(op[0]) * o_ferm[0][n]])
 
-    o_qubit = simplify(qml.Hamiltonian(coeffs, ops), cutoff=cutoff)
+    return simplify(qml.Hamiltonian(coeffs, ops), cutoff=cutoff)
 
-    return o_qubit
+
+@lru_cache
+def get_pauli_map(n):
```

Comment:
This should be a private function?

Reply:
You mean naming it `_get_pauli_map`? Yes probably. I think we could even move it to `pennylane.pauli.utils`, no?

Reply:
Yes, it can be renamed to `_get_pauli_map` and moved to `pennylane.pauli.utils`.